### DISCUSSION TITLE
🚀 Start UI [web] now with React 18 and Chakra UI v2

### DISCUSSION THREAD

**Discussion by yoannfleurydev** [BearStudio/start-ui-web#223] [Announcements]
🚀 Start UI [web] is now using [React 18](https://reactjs.org/blog/2022/03/29/react-v18.html) ([Next.JS documentation](https://nextjs.org/docs/advanced-features/react-18/overview)) and [Chakra UI v2](https://chakra-ui.com/guides/migration)

Thanks for following and using this project. Feel free [to share this project](https://twitter.com/intent/tweet?text=Check%20the%20awesome%20%40startui_%20project%20on%20%40github!%20%23React%20%23ChakraUI%20%23Nextjs&url=https%3A%2F%2Fgithub.com%2FBearStudio%2Fstart-ui-web) or this discussion on your social networks, it helps 🚀 Start UI to grow.

<img width="1280" alt="GitHub Start UI  web  - React 18 - Chakra v2 (2)" src="https://user-images.githubusercontent.com/3920615/170794887-a6675ba4-55dc-4f7e-8c63-8efd72994a15.png">

**Comment by vanenshi** [comment]
Thank you for your great work
